src/constants/config.json

```diff
@@ -7017,6 +7017,41 @@
 					"packageName": "elevate-self-creation-portal"
 				}
 			]
+		},
+		{
+			"sourceRoute": "/interface/v1/internal/fetchResources",
+			"type": "POST",
+			"priority": "MUST_HAVE",
+			"inSequence": false,
+			"orchestrated": true,
+			"responseMessage": "Resource fetched Successfully",
+			"rateLimit": {
+				"type": "internal"
+			},
+			"targetPackages": [
+				{
+					"basePackageName": "project",
+					"packageName": "elevate-project",
+					"targetBody": [],
+					"responseBody": [
+						{
+							"sourceField": "data",
+							"targetField": "data[]"
+						}
+					]
+				},
+				{
+					"basePackageName": "samiksha",
+					"packageName": "elevate-survey",
+					"targetBody": [],
+					"responseBody": [
```

Comment:
@adithyadinesh0412  why we need to do this "targetField": "data[]"

Reply:
This is the method to combine the results in the interface service by design. @priyanka-TL